units/host_monitoring_external_termination.go

```diff
@@ -92,58 +92,66 @@ func (j *hostMonitorExternalStateCheckJob) Run(ctx context.Context) {
 		j.env = evergreen.GetEnvironment()
 	}
 
-	settings := j.env.Settings()
+	_, err = CheckExternallyTerminatedHost(ctx, j.ID(), j.env, j.host)
+	j.AddError(err)
+}
 
-	cloudHost, err := cloud.GetCloudHost(ctx, j.host, settings)
+func CheckExternallyTerminatedHost(ctx context.Context, id string, env evergreen.Environment, h *host.Host) (bool, error) {
```

Comment:
Document this function, especially what the boolean means. It might be clearer to call this `IsExternallyTerminated`, but I don't have strong feelings about that.

Reply:
done.
I changed the name to `HandleExternallyTerminatedHost` to indicate the function changes the state of the host in the database.